pkg/authorization/client/clientset_generated/internalclientset/clientset.go

```diff
@@ -0,0 +1,73 @@
+package internalclientset
+
+import (
+	"github.com/golang/glog"
+	v1core "github.com/openshift/origin/pkg/authorization/client/clientset_generated/internalclientset/typed/core/v1"
+	restclient "k8s.io/kubernetes/pkg/client/restclient"
+	discovery "k8s.io/kubernetes/pkg/client/typed/discovery"
+	"k8s.io/kubernetes/pkg/util/flowcontrol"
+)
+
+type Interface interface {
+	Discovery() discovery.DiscoveryInterface
+	Core() v1core.CoreInterface
```

Comment:
It's effectively the inner interface, and was convention "decided" upstream.

On Jun 16, 2016, at 9:09 AM, David Eads <notifications@github.com> wrote:

In
pkg/authorization/client/clientset_generated/internalclientset/clientset.go
<https://github.com/openshift/origin/pull/9372#discussion_r67340417>:

> @@ -0,0 +1,73 @@
> +package internalclientset
> +
> +import (
> +	"github.com/golang/glog"
> +	v1core "github.com/openshift/origin/pkg/authorization/client/clientset_generated/internalclientset/typed/core/v1"
> +	restclient "k8s.io/kubernetes/pkg/client/restclient"
> +	discovery "k8s.io/kubernetes/pkg/client/typed/discovery"
> +	"k8s.io/kubernetes/pkg/util/flowcontrol"
> +)
> +
> +type Interface interface {
> +	Discovery() discovery.DiscoveryInterface
> +	Core() v1core.CoreInterface

This looks strange to me. Core?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/9372/files/2a0ceea3c7e61fc7e21b9e5471a651781638e7d5#r67340417>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABG_p9QklkgM94JMGs_2obbASJ_0FiWzks5qMUsRgaJpZM4I3W73>
.